### PR TITLE
Added the ability to get guild data directly from the ID

### PR DESCRIPTION
**Pull request by Mythbusters123**

### CODE REVIEW

File: routes/spec.js [hunk 1]
```diff
@@ -1003,6 +1003,162 @@ Consider supporting The Slothpixel Project on Patreon to help cover the hosting
...
+                      },
+                    },
+                  },
+                },
+              },
+            },
+          },
+        },
+        route: () => '/guilds/id/:id',
+        func: async (request, response, callback) => {
```
Comment (builder-247) [inline routes/spec.js:1149]: Can you also add the `populatePlayers` parameter? https://github.com/slothpixel/core/blob/master/routes/spec.js#L839

Comment (builder-247) [reply routes/spec.js:1149]: Also you need to add the parameter to the parameters array so it shows up on docs correctly

Comment (Mythbusters123) [reply routes/spec.js:1149]: I will when I get home :)

Comment (Mythbusters123) [reply routes/spec.js:1149]: I already did that.

Comment (builder-247) [reply routes/spec.js:1149]: On lines 1015 and 859

Comment (Mythbusters123) [reply routes/spec.js:1149]: They are already there, unless something is messed up

Comment (builder-247) [reply routes/spec.js:1149]: Are you sure you've commited them? I can't see them on github

Comment (Mythbusters123) [reply routes/spec.js:1149]: thats weird, lemme check github

**Review by builder-247 — APPROVED** [review]

**Pull request merged**